openaps/oref0
# re-add openaps first-upload (mentioned in docs)

To test, follow https://openaps.readthedocs.io/en/latest/docs/Customize-Iterate/update-your-rig.html#alternative-step-1b-test-a-feature-branch-not-recommended-for-initial-setup to checkout the `first-upload` branch.

Things to test:
 - Make sure that `openaps first-upload` works
 - Set pump clock to be more than 5m apart from the rig clock, and make sure oref0-pump-loop automatically resets it
